includes/class-solrpower-sync.php

```
@@ -346,8 +349,10 @@ function delete_all() {
 				$update->addCommit();
 				$solr->update( $update );
 			}
+			return true;
 		} catch ( Exception $e ) {
-			echo esc_html( $e->getMessage() );
+			//echo esc_html( $e->getMessage() );
```

What's the rationale for swallowing this error?